docs/cli.md

```diff
@@ -43,7 +43,7 @@ sb deploy [--docker-image]
 
 | Name | Default | Description |
 | --- | --- | --- |
-| `--docker-image` `-i` | `superbench/superbench` | Docker image URI. |
+| `--docker-image` `-i` | `superbench/superbench` | Docker image URI, [here](/docs/tutorial/container-images) listed all images. |
```

Comment:
This link is for website, doesn't work in github?

Reply:
updated to relevant link